OvmfPkg/AcpiPlatformDxe/Xen.c

```diff
@@ -56,10 +58,12 @@ XenDetected (
   @return EFI_NOT_FOUND      Don't find Xen RSDP structure.
   @return EFI_ABORTED        Find Xen RSDP structure, but it's not integrated.
 
-**/
+**/ 
 EFI_STATUS
 EFIAPI
-GetXenAcpiRsdp (
+FindRsdp (

```

Comment:
Instead of making further changes to Xen.c, I think we should keep this file untouched and it's time to create our own InstallAcrnTables with GetAcrnAcpiRsdp in OvmfPkg/AcpiPlatformDxe/Acrn.c. Can you make these changes?